addons/binding/org.openhab.binding.yeelight/ESH-INF/thing/ceiling.xml

```diff
@@ -12,11 +12,7 @@
 			<channel id="colorTemperature" typeId="colorTemperature" />
 		</channels>
 
-		<config-description>
-			<parameter name="deviceId" type="text" required="true">
-				<label>Device ID</label>
-				<description>Id of the Yeelight device to connect with.</description>
-			</parameter>
-		</config-description>
+		<config-description-ref uri="thing-type:device:base" />
```

Comment:
I don't quite understand what this does.

Reply:
Alright. The problem is, I cannot find this file that is referred to. Also I guess this file would need a section for the duration parameter, so should have been changed by this PR. I cannot see it in the changed files, though.

Reply:
Good catch. Should be there now.